src/sentry/static/sentry/app/views/eventsV2/landing.tsx

```diff
@@ -373,7 +408,7 @@ const StyledSearchBar = styled(SearchBar)`
 const StyledActions = styled('div')`
   display: grid;
   grid-gap: ${space(2)};
-  grid-template-columns: auto min-content;
+  grid-template-columns: auto max-content min-content;
```

Comment:
Does this need a different layout in mobile/tablet viewports?

Reply:
I made these stack instead on smaller viewports:
<img width="478" alt="image" src="https://user-images.githubusercontent.com/4205004/99104141-989e9900-25ae-11eb-98fb-5faa8a1c0019.png">